### DISCUSSION TITLE
How to get notifications for expiry of keys in Jetstream KV store

### DISCUSSION THREAD

**Discussion by UltraNemesis** [nats-io/nats.go#1629] [Q&A]
As the title suggests, looking for a way to get notification of expiry for Jetstream kv store. Watcher doesn't seem to provide it and I also tried putting a subscription on "$KV.store_name.>" and that didn't help either. I get notifications for adds and changes only.

**Comment by ripienaar** [comment]
At the moment this is not supported, we plan to add it in the 2.11 or 2.12 releaes.
